FirebaseMessaging/Sources/Public/FIRMessaging.h

```diff
@@ -504,6 +536,20 @@ NS_SWIFT_NAME(Messaging)
  */
 - (FIRMessagingMessageInfo *)appDidReceiveMessage:(NSDictionary *)message;
 
+#pragma mark - GDPR
+/**
+ * Deletes all the tokens and checkin data of the Firebase project  and related data on the server
+ * side. A network connection is required for the method to succeed.
+ *
+ * This does not delete the Firebase Installations ID that may have been created when
+ * generating the token. See `FIRInstallations deleteWithCompletion:` for deleting that.
+ *
+ * @param completion A completion handler which is invoked when the operation completes. `error ==
+ * nil` indicates success.
+ */
+- (void)deleteWithCompletion:(void (^)(NSError *__nullable error))completion
+    NS_SWIFT_NAME(delete(completion:));
```

Comment:
I think `NS_SWIFT_NAME` is redundant here:
![Screen Shot 2020-09-01 at 4 59 35 PM](https://user-images.githubusercontent.com/1841926/91905442-8cc05d80-ec74-11ea-910d-b684c1435548.png)

Reply:
I notice this new swift header file, is it a new thing?
I guess with this, we don't need to define any of the above NS_SWIFT_NAME anymore if it's by default defined here?
also add @paulb777 and @ryanwilson here

Reply:
Compiler generates Swift API for Objective-C. The result usually looks pretty well for Objective-C API that follows guidelines. [NS_SWIFT_NAME](https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/renaming_objective-c_apis_for_swift) is needed only when renaming is required for some reason. 

You may preview the generated Swift interface for any .h file with Xcode using the editor feature as shown on the screenshot:

<img width="513" alt="Screen Shot 2020-09-01 at 6 21 43 PM" src="https://user-images.githubusercontent.com/1841926/91912494-46bdc680-ec81-11ea-81e7-959e00d232d1.png">